src/couch/src/couch_lru.erl

```diff
@@ -16,33 +16,34 @@
 -include("couch_server_int.hrl").
 
 new() ->
-    {gb_trees:empty(), dict:new()}.
+    % {gb_trees(UniqueInt -> DbName), #{DbName => UniqueInt}}
+    {gb_trees:empty(), #{}}.
 
-insert(DbName, {Tree0, Dict0}) ->
+insert(DbName, {Tree0, #{} = Dict0}) ->
```

Comment:
Maybe rename `Dict` vars to `Map`?

Reply:
That would work, good idea. I thought about it, but couldn't decide thinking a "dict" is technically is not incorrect in the general sense. But, yeah it would be less confusing if the name matched the data type.

Reply:
I think I'll just create a separate PR which updates the dict to map only  and also adds some units tests. This way we'll separate the two things and if we make any logic LRU updates we'll have tests to run at least.

Reply:
I wonder if some `-spec` statements on the public functions might also be helpful :)

Reply:
I did so in this PR: https://github.com/apache/couchdb/pull/5168